### ISSUE TITLE
How to draw both training loss and validation loss in one map chart

### ISSUE THREAD

**Issue by duongdqq** [AlexeyAB/darknet#6487]
Is there anyone try to draw both training loss and validation loss in one map chart
Thank for for reading
